gateway-ha/src/test/java/io/trino/gateway/ha/clustermonitor/TestClusterStatsMonitor.java

```diff
@@ -61,6 +62,12 @@ void testJdbcMonitor()
         testClusterStatsMonitor(backendStateConfiguration -> new ClusterStatsJdbcMonitor(backendStateConfiguration, new MonitorConfiguration()));
     }
 
+    @Test
```

Comment:
add tests for the failure cases, 400 and 500 response codes with non json and invalid json response bodies